README.md

```diff
@@ -28,6 +28,8 @@
       * Alternatively, you can also compile `oc` from source.
       * Note: It is recommended to put the `oc` binary somewhere in your path.
     * Mac OSX: [Minishift](https://github.com/minishift/minishift/releases) is the recommended development environment
+  * Create a `kubectl` symlink to the `oc` binary (Note: It is recommended to put the `kubectl` symlink somewhere in your path.):
+    * `ln -s oc kubectl`
```

Comment:
What is the rationale behind this one?

Reply:
If you don't have it, then the playbook fails (at least it did for @joelddiaz and me).

@joelddiaz said it's specifically the `kubectl_apply` module requires it.